modules/loader/src/main/scala/com/snowplowanalytics/snowplow/rdbloader/loading/Load.scala

```diff
@@ -146,11 +146,10 @@ object Load {
                       inTransactionMigrations *>
                       run[F, I](setLoading, discovery.discovery, initQueryResult, target, disableMigration).flatMap {
                         loadedRecoveryTableNames =>
-                          setStage(Stage.Committing) *>
-                            Manifest.add[F](discovery.origin.toManifestItem) *>
-                            Manifest
-                              .get[F](discovery.discovery.base)
-                              .map(opt => LoadSuccess(opt.map(_.ingestion), loadedRecoveryTableNames))
+                          for {
+                            _ <- setStage(Stage.Committing)
+                            _ <- Manifest.add[F](discovery.origin.toManifestItem)
+                          } yield LoadSuccess(loadedRecoveryTableNames)
```

Comment:
Drawing attention to this change.  Previously, we did an extra lookup to the manifest table, and the only reason was to get a timestamp (essentially "now").  After this change we simply use the loader's local version of when is "now".  The value is used in monitoring payloads.

Arguably it doesn't belong in this PR, which is supposed to be about adding a statsd metric for recovery tables.

But.... it is related to how we pass around data relating to the load.  Data which is used for building metrics and monitoring payloads.  So I smuggled it into this PR.